pymongo/helpers.py

```diff
@@ -72,6 +72,8 @@ def _index_list(key_or_list, direction=None):
     else:
         if isinstance(key_or_list, str):
             return [(key_or_list, ASCENDING)]
+        if isinstance(key_or_list, abc.ItemsView):
+            return list(key_or_list)
```

Comment:
Why is this change needed?

Reply:
This is necessary so that when a dict_items object is passed into _index_list it will return a list rather than a dict_items object.